tests/runtest

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env lua
 
 package.path = "../?.lua;"..package.path
-require "fun" ()
+fun = require "fun"
```

Comment:
One another pure cosmetic change. Please, discard.

Reply:
nope. it's assigned to global